glib/src/subclass/object.rs

```diff
@@ -612,7 +609,7 @@ mod test {
                     .to_value(),
             )
         });
-        let value = obj.emit_by_name("create-child-object", &[]).unwrap();
+        let value: glib::Object = obj.emit_by_name("create-child-object", &[]);
```

Comment:
```suggestion
        let value = obj.emit_by_name::<glib::Object>("create-child-object", &[]);
```
Any reason why this one doesn't use the turbo-fish annotation?

Reply:
In this one I wanted to check if type inference of the return type works, that's why :)